testsuite/tests/typing-unique/unique.ml

```diff
@@ -340,26 +349,29 @@ Line 1, characters 14-20:
 Error: Unbound value "update"
 |}]
 
-let inf1 (unique_ x : float) = unique_ let y = x in y
+let inf1 (unique_ x : non_mode_cross) = unique_ let y = x in y
 [%%expect{|
-val inf1 : float @ unique -> float = <fun>
+val inf1 : non_mode_cross @ unique -> non_mode_cross = <fun>
 |}]
 
-let inf2 (b : bool) (unique_ x : float) = unique_ let y = if b then x else 1.0 in y
+let inf2 (b : bool) (unique_ x : non_mode_cross) = unique_ let y = if b then x else [] in y
```

Comment:
Iiuc `[]` should be coaliased someday, or something like that, so why not instead use `mk_non_mode_cross ()`?